lib/arclight/traject/ead2_config.rb

```diff
@@ -275,7 +275,21 @@
   end
 
   to_field 'access_subjects_ssm' do |_record, accumulator, context|
-    accumulator.concat Array.wrap(context.output_hash['access_subjects_ssim'])
+    accumulator.concat(context.output_hash.fetch('access_subjects_ssim', []))
+  end
+
+  # Indexes the acquisition group information into the notes field
+  # Please see https://www.loc.gov/ead/tglib/elements/acqinfo.html
+  to_field 'acqinfo_ssim', extract_xpath('/xmlns:ead/xmlns:archdesc/xmlns:acqinfo/xmlns:note')
```

Comment:
I'm not certain why, but simply using `./xmlns:acqinfo/xmlns:note` and `xmlns:acqinfo/xmlns:note` were specific to the current component.  Initially just `/xmlns:ead/xmlns:archdesc/xmlns:acqinfo` worked, but I then noticed that the fixtures did not offer any cases where `acqinfo` is a child of anything other than `archdesc` (https://github.com/projectblacklight/arclight/pull/602/commits/d4e55403a642cf22544d538a5bedb24f35766619 rectifies this).

Please let me know how these XPaths can be improved.